.babelrc

```diff
@@ -1,12 +1,33 @@
 {
-  "presets": ["airbnb"],
-  "plugins": [
-    "inline-react-svg",
-    ["transform-replace-object-assign", "object.assign"],
-  ],
   "env": {
     "test": {
-      "plugins": ["istanbul"]
-    }
-  }
+      "presets": ["airbnb"],
+      "plugins": [
+        "inline-react-svg",
```

Comment:
super gnarly that we have to repeat these two plugins in every env due to ordering issues